### PR TITLE
- Add samples indicies

### PR DESCRIPTION
**Pull request by sptramer**
- Update document metadata

### CODE REVIEW

**Comment by sptramer** [comment]
@joshgav Please sign off on this as good for the site - the format mirrors what's available for other languages currently, and I have some suggestions based on this format that may help guide our approach to samples in general.

File: azure-go-sdk-conceptual/azure-sdk-go-auth-aad-samples.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ms.date: 03/21/2018
+ms.topic: article
+ms.devlang: go
+manager: carmonm
+---
+
+# Azure SDK for Go samples for authentication and AAD
+
+The following table links to selected samples of Go source code that you can use to authenticate with the Azure SDK for Go and work with Azure Activce Directory (AAD) services. To get all of the sample code and tests for authentication and AAD, use the `go get` command:
+
```
Comment (joshgav) [inline azure-go-sdk-conceptual/azure-sdk-go-auth-aad-samples.md]: I think `git clone` would be better here, cause we don't intend for users to link in the sample code in apps. In fact our goal is mainly to give them something to read and copy, so perhaps we don't need any hints on how to download?

File: azure-go-sdk-conceptual/azure-sdk-go-auth-aad-samples.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+```bash
+go get -d -u github.com/azure-samples/azure-sdk-for-go-samples/iam
+go get -d -u github.com/azure-samples/azure-sdk-for-go-samples/authorization
+go get -d -u github.com/azure-samples/azure-sdk-for-go-samples/graphrbac
+```
+
+All samples for the Azure SDK for Go are available on [GitHub](https://github.com/Azure-Samples/azure-sdk-for-go-samples).
+
+| Name | Description |
```
Comment (joshgav) [inline azure-go-sdk-conceptual/azure-sdk-go-auth-aad-samples.md]: Nice!

File: azure-go-sdk-conceptual/azure-sdk-go-compute-networking-samples.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ms.author: sttramer
+ms.date: 03/21/2018
+ms.topic: article
+ms.devlang: go
+manager: carmonm
+---
+
+# Azure SDK for Go samples for compute and networking
+
+The following table links to selected samples of Go source code that you can use to manage VMs, virtual networks, and subnets in Azure. To get all of the sample code and tests for compute and networking, use the `go get` command:
```
Comment (joshgav) [inline azure-go-sdk-conceptual/azure-sdk-go-compute-networking-samples.md]: Same as above, let's not mention download or consider `git clone` instead.

File: azure-go-sdk-conceptual/azure-sdk-go-containers-samples.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ms.author: sttramer
+ms.date: 03/21/2018
+ms.topic: article
+ms.devlang: go
+manager: carmonm
+---
+
+# Azure SDK for Go samples for containers
+
+The following table links to selected samples of Go source code that you can use to work with containers on Azure via Azure Container Instances (ACI) or Azure Kubernetes Service (AKS). To get all of the sample code and tests for containers, use the `go get` command:
```
Comment (joshgav) [inline azure-go-sdk-conceptual/azure-sdk-go-containers-samples.md]: as above

File: azure-go-sdk-conceptual/azure-sdk-go-sql-database-samples.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+ms.date: 03/21/2018
+ms.topic: article
+ms.devlang: go
+manager: carmonm
+---
+
+# Azure SDK for Go samples for Azure SQL Database
+
+The following table links to selected samples of Go source code that you can use to work with Azure SQL Database. To get all of the sample code and tests for SQL, use the `go get` command:
+
```
Comment (joshgav) [inline azure-go-sdk-conceptual/azure-sdk-go-sql-database-samples.md]: as above

**Pull request merged**